src/icon-manifest/supportedExtensions.ts

```diff
@@ -476,6 +476,7 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    { icon: 'stylable', extensions: ['.st.css'], languages: [languages.stylable], format: FileFormat.svg },
```

Comment:
Also, remove the `.st.css` from the `extensions` attribute. Using `language id` ensures that whatever extensions are specified in the vscode `stylable` extension will be used. And in this case this [extension](https://marketplace.visualstudio.com/items?itemName=wix.stylable-intelligence) has it defined @  https://github.com/wix/stylable-intelligence/blob/master/packages/client/package.json#L33

Reply:
Just to make sure I understood, by using our language-intelligence extension, and allowing it to determine the language of the file - vscode-icons will mark files with the Stylable icon on any file determined to be a Stylable stylesheet (by said language-intelligence)?

Reply:
Yes. The icon will lit as soon as the extension get's installed on `vscode`.  And any file extension defined in the 3rd party extension will be automatically supported. That's the power of the `language id`.